### DISCUSSION TITLE
Where to specify ansible user

### DISCUSSION THREAD

**Discussion by wannesrams** [HCL-TECH-SOFTWARE/connections-automation#7] [Q&A] [answered]
Hi, I cannot find the standard formatted inventory yaml file where I could just use ansible_user= to do this. I am running ansible of my mac so it defaults to my mac user trying to execute the scripts on the remote host. That obviously fails

**Comment by dejo1307** [comment, marked as answer]
Hi! Just add -u your_ansible_user after the command, and it will instruct Ansible to execute as your_ansible_user remotely.

**Reply by wannesrams** [reply]
Thank you Dejan

**Comment by stoeps13** [comment]
You can use `ansible.cfg` and set it directly in the root of the project, so you can't forget to add it on the command line. Explained here: https://github.com/HCL-TECH-SOFTWARE/connections-automation/pull/13
